### PR TITLE
update docs with example usage

### PR DESCRIPTION
**Pull request by hassan-yonomi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,3 +112,62 @@ Add a listener for request
...
+Remove a Child device
+
+```
+client.relatedDevices.remove({deviceId}, {deviceType}, {physicalId})
+```
+
+Update Related Device State (publish)
+
+```
+client.relatedDeivcesMap[{deviceId}].request.publish({method}, {params})
```
Comment (stewartjarod) [inline README.md]: `relatedDeivcesMap` sp `Devices`

File: README.md [hunk 2]
```diff
@@ -112,3 +112,62 @@ Add a listener for request
...
+```
+
+Update Related Device State (RPC)
+
+```
+client.relatedDeivcesMap[{deviceId}].request.rpc({method}, {params})
```
Comment (stewartjarod) [inline README.md]: ditto

File: README.md [hunk 3]
```diff
@@ -112,3 +112,62 @@ Add a listener for request
...
+Get Related device Ids from cloud
+
+```
+client.relatedDeviceManager.getRelatedDeviceIds()
+```
+
+Get Related device
+
+```
+client.relatedDeviceManager.getRelatedDevice({Id})
```
Comment (stewartjarod) [inline README.md]: `getRelatedDevices` vs `getRelatedDevice` ?

Comment (hassan-yonomi) [reply README.md]: it gets a singe device by id

File: README.md [hunk 4]
```diff
@@ -112,3 +112,62 @@ Add a listener for request
 ```
 client.eventSource.on('requests', function (err, res))
 ```
+
+
+### Managing Child/Related Devices
+
+Related Device accessor
```
Comment (varddum) [inline README.md]: nit: These look like they can be level 4 headers.

File: README.md (hunk 3 above)
Comment (stewartjarod) [reply README.md]: 👍

File: README.md (hunk 4 above)
Comment (hassan-yonomi) [reply README.md]: what are those ?

**Review by mikelittman — APPROVED** [review]

**Review by stewartjarod — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -112,3 +112,62 @@ Add a listener for request
...
+client.relatedDeivcesMap[{deviceId}].request.publish({method}, {params})
+```
+
+Update Related Device State (RPC)
+
+```
+client.relatedDeivcesMap[{deviceId}].request.rpc({method}, {params})
+```
+
+Related Device Management
```
Comment (varddum) [inline README.md]: The level 3 header makes this look unnecessary.

File: README.md (hunk 4 above)
Comment (mikelittman) [reply README.md]: I think he just means go deeper `####` vs `###`

Comment (stewartjarod) [reply README.md]: Prolly a good idea

**Pull request merged**